### ISSUE TITLE
Add support for LLVM 14

### ISSUE THREAD

**Issue by atlx** [TheDan64/inkwell#339]
I have LLVM 14 installed and when building, it's not picking it up. I tried changing my `Cargo.toml` to `llvm14-0`:

```toml
inkwell = { git = "https://github.com/TheDan64/inkwell", branch = "master", features = ["llvm14-0"] }
```
But I get the following:
```
the package `my_package` depends on `inkwell`, with features: `llvm14-0` but `inkwell` does not have these features.
```
Seems like LLVM 14 support is yet to be added?

**Comment by TheDan64** [comment]
LLVM 14 is already supported, I'm not really sure why you're running into this. Feature is right here: https://github.com/TheDan64/inkwell/blob/master/Cargo.toml#L31

**Comment by seanyoung** [comment]
Can we have a published crate with LLVM 14 support please? We need this upgrade to LLVM 14 in [Hyperledger Solang](https://github.com/hyperledger/solang). Thanks!

**Comment by TheDan64** [comment]
@seanyoung we'll be releasing a 0.1.0 version in the near future which will include it. There a couple remaining issues outstanding

**Comment by seanyoung** [comment]
The last crate version was published a year ago. Publishing a crate is cheap and this is causing real pain. Why can't you just publish another 0.1.0-beta.5?

TBH I've been looking at ripping inkwell out of my project, this is just unbearable.

**Comment by TheDan64** [comment]
Sure thing, can do. Going to close out this ticket as LLVM 14 is already supported

**Comment by TheDan64** [comment]
Done

**Comment by TheDan64** [comment]
@seanyoung fyi, just published 0.1.0 w/ LLVM15. Thank you for your patience and understanding

**Comment by seanyoung** [comment]
Wonderful, thank you very much.
